test/ng/directive/ngOptionsSpec.js

```diff
@@ -779,6 +779,23 @@ describe('ngOptions', function() {
     expect(options[2]).not.toBeMarkedAsSelected();
   });
 
+
+  it('should render the options only one time', function() {
+    scope.values = ['black', 'white', 'red'];
+    createSelect({
+      'ng-model': 'value'
+    });
+
+    element.attr('ng-options', 'value for value in values');
```

Comment:
you should pass the ngoptions expression like you did with ng-model. Or use createSingleSelect which has useful defaults.